web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -181,10 +182,6 @@ private boolean invokeSafeOnBeginRequest(ServletRequest req, ServletResponse res
     private void invokeSafeOnEndRequest(ServletRequest req, ServletResponse res) {
         try {
             webModulesContainer.invokeOnEndRequest(req, res);
```

Comment:
This call invokes the modules' onEndRequest method which uses the TLS. After cleanup above, the TLS would no longer be available